tikv-configuration-file.md

```diff
@@ -143,24 +143,24 @@ TiKV 配置文件比命令行参数支持更多的选项。你可以在 [etc/con
 ### `use-unified-pool`
 
 + 是否使用统一的读取线程池（在 [`readpool.unified`](#readpoolunified) 中配置）处理存储请求。该选项值为 false 时，使用单独的存储线程池。通过本节 (`readpool.storage`) 中的其余配置项配置单独的线程池。
-+ 默认值：true
++ 默认值：false
```

Comment:
I think it's `true` in 5.0.

Reply:
I check tikv code in src/config.rs, seems it is set to None. I assume None is false?
![image](https://user-images.githubusercontent.com/26134513/110075188-61eac980-7dbd-11eb-9ab0-c221dce12a40.png)

Reply:
Ah, It's a bit complicated. It should be changed to be the same as the coprocessor one:
```
默认值：如果本节 (`readpool.storage`) 中没有其他配置，默认为 true。否则，为了升级兼容性，默认为 false，请根据需要更改 [`readpool.unified`](#readpoolunified) 中的配置后再启用该选项。
```